ios/ScannerBle.swift

```diff
@@ -89,7 +61,10 @@ class BeaconScanner: NSObject, CBCentralManagerDelegate {
     guard isAdvertisingURL(serviceData) else { return }
     
     if let url = URLForFrame(serviceData) {
-      self.addURL(url)
+      callback([
+        "url": url.absoluteString,
+        "distance": -1
```

Comment:
Yeah, that is a missing feature of the ios library and should be introduced as a followup